spec/npmdc_spec.rb

```diff
@@ -195,4 +195,18 @@
       expect { subject }.to write_output(output_msg)
     end
   end
+
+  context 'success scoped packages check' do
```

Comment:
Hm. Versions checking didn't change and we already have failure spec for it. This PR change only traversing of `node_modules`. What do you want to test in the failure spec?